Calificativ.js

```diff
@@ -15,3 +15,22 @@
      1: IS
 */
 
+
+const miorita = {
```

Comment:
Cred că aici ai copiat codul greșit. Ai dat tema cu Miorița în ambele fișiere.

Reply:
cred ca le-am incurcat, era in tema refacuta.
acum mi-am dat seama ca ar fi trebuit pus inca un else if intre 6 si 0. 

function calificative(nota) {
    if (nota > 8 && nota <11) {
        return("FB");
    }
    else if (nota > 6 && nota < 9) {
        return("B");
    }
    else if (nota > 4 && nota <7) {
        return("S");
    }
    else {
        return("IS") 
    }